pkg/model/openstackmodel/servergroup.go

```diff
@@ -152,6 +152,8 @@ func (b *ServerGroupModelBuilder) buildInstances(c *fi.ModelBuilderContext, sg *
 			Metadata:         igMeta,
 			SecurityGroups:   ig.Spec.AdditionalSecurityGroups,
 			AvailabilityZone: az,
+			ClusterName:      s(b.ClusterName()),
+			GroupName:        s(ig.Name),
```

Comment:
Do we need the group on the instance if we have a reference to the ServerGroup which contains the instance group name?